tech/languages/csharp/dotnet-installation.md

```diff
@@ -4,18 +4,29 @@ subsection: csharp
 order: 3
 ---
 
-# .NET SDK installation
+# .NET Core installation
 
-.NET Core is still work in progress, and below download locations contain packages that are not entirely _free._ Please be patient, we should have reasonable packages within a few months.
+## Fedora .NET SIG repository
 
-Choose based on your Fedora and SDK version:
+.NET Core is still work in progress. The download location below contains packages that do not meet the official Fedora requirements yet (such as containing prebuilt binaries or bundled dependencies.) Please be patient, we should have reasonable packages within a few months.
```

Comment:
There is only our copr and Microsoft as source of these packages, where the Microsoft build does not follow upstream agreed conventions (as usual..) but they're both up to date and from almost the same source code. Ours is built from the same code as RHEL which may or may not contain an extra RHEL specific fix or two... @omajid would know more about that, but anyway, I don't think that there is anything that would be "outdated." [nmilosev](https://copr.fedorainfracloud.org/coprs/nmilosev/) has a few obsolete copr repos that are marked as obsolete, and should be removed soon ™️